src/App.jsx

```diff
@@ -1,13 +1,15 @@
 import { useState } from "react";
-import Navbar from "./components/Navbar";
-import PostFeed from "./components/PostFeed";
+import Article from "./pages/Article";
+import Home from "./pages/Home";
 import { Route, Routes } from "react-router";
+import Error from "./pages/Error";
 function App() {
   return (
     <>
-      <Navbar />
       <Routes>
-        <Route path="/" element={<PostFeed />} />
+        <Route path="/" element={<Home />} />
+        <Route path="/home" element={<Home />} />
+        <Route path="/articles/:topic/:article_id/comments" element={<Article />} />
```

Comment:
this is quite a long route to take the user to the individual article! this URL also suggests that the user is only going to be looking at the comments of this article rather than the whole article itself. I would end the URL before /comments if you'd like to keep the topic in there